src/main/scala/loamstream/model/jobs/JobOracle.scala

```diff
@@ -20,6 +21,10 @@ trait JobOracle {
 }
 
 object JobOracle {
```

Comment:
Just curious if `Oracle` here refers to the company or an "all seeing" object/system?

Reply:
The latter.